src/scheduling.py

```diff
@@ -85,6 +95,16 @@ def set_role_priority():
             if new_priority is not None:
                 role.priority = float(new_priority)
         database.flush()
+        return redirect("/welcome")
+
+    @app.route("/restaurant/<restaurant_id>")
```

Comment:
This looks like a merge conflict. This method was moved down lower in the file. I think this instance can be deleted and update the version around line 183

Reply:
Can you elaborate on that.

Reply:
I may have been looking at an old diff. Looking at your fork this has been corrected.